src/Symfony/Component/Console/Application.php

```diff
@@ -265,18 +265,27 @@ public function doRun(InputInterface $input, OutputInterface $output)
         return $exitCode;
     }
 
+    /**
+     * @deprecated since version 4.2, to be removed in 5.0. Use individual helpers instead.
+     */
     public function setHelperSet(HelperSet $helperSet)
     {
+        @trigger_error('setHelperSet() is deprecated since version 4.2 and will be removed in 5.0. Use individual helpers instead.', E_USER_DEPRECATED);
```

Comment:
`Method %s is deprecated since Symfony 4.2`, `%s` being `__METHOD__`. And no need for the `and` part

Reply:
`Method "%s()" ...` actually :)

Reply:
`sprintf('Method "%s()" is deprecated since version 4.2, use individual helpers instead.', __METHOD__)`

Reply:
I followed the contribution guidelines, maybe change those as well then?